### ISSUE TITLE
istio v0.8 tasks/security/basic authentication docs

### ISSUE THREAD

**Issue by jsenon** [istio/istio#5909]
istio version: 

`istio-release-0.8-20180520-18-17`

Trying to apply, on my own namespace:

```cat <<EOF | istioctl create -f -
apiVersion: "networking.istio.io/v1alpha3"
kind: "DestinationRule"
metadata:
  name: "enable-mtls"
  namespace: "mtls"
spec:
  host: *.mtls.svc.local.cluster
  trafficPolicy:
    tls:
      mode: ISTIO_MUTUAL
EOF
```

But I receive error message:

`Error: cannot parse proto message: error converting YAML to JSON: yaml: line 7: did not find expected alphabetic or numeric character`

If I apply:

```cat <<EOF | istioctl create -f -
apiVersion: "networking.istio.io/v1alpha3"
kind: "DestinationRule"
metadata:
  name: "enable-mtls"
  namespace: "mtls"
spec:
  host: "*.mtls.svc.local.cluster"
  trafficPolicy:
    tls:
      mode: ISTIO_MUTUAL
EOF
```

I receive:

`Error: cannot parse proto message: YAML decoding error: host: '*.mtls.svc.local.cluster'
trafficPolicy:
  tls:
    mode: ISTIO_MUTUAL
 unknown value "ISTIO_MUTUAL" for enum istio.networking.v1alpha3.TLSSettings_TLSmode`

**Comment by jsenon** [comment]
My bad, could you confirm that we could only have one destination rules? If yes, how can we set multiple host?

**Comment by sakshigoel12** [comment]
cc. @wattli

**Comment by yangminzhu** [comment]
I think this is fixed in Istio 0.8 release.
@jsenon, Could you try to update your Istio version to the official 0.8 version? (See https://istio.io/docs/setup/kubernetes/quick-start/#download-and-prepare-for-the-installation)

Thank you!
